### ISSUE TITLE
Convert checkpoint to Diffusers checkpoint

### ISSUE THREAD

**Issue by tuanh123789** [gligen/GLIGEN#49]
Great Work!
I see that with checkpoint text box and inpainting text to can use in Diffusers library and I want to use other checkpoint with Diffusers too. How can I convert your original checkpoint to Diffusers checkpoint.
Thank you !

**Comment by k15201363625** [comment]
+1

**Comment by tuanh123789** [comment]
> +1

https://github.com/huggingface/diffusers/blob/main/scripts/convert_gligen_to_diffusers.py I have implement this and summit it on diffusers. You can check this

**Comment by zengjie617789** [comment]
@tuanh123789  hello, I wonder that the netework of GLIGEN is different from the diffusers and how to convert it successfully?
As far as i know, it added new parameters in that, after converted, how to use them in diffusers?
>GLIGEN: The way GLIGEN introduces new parameters is by adding a gated self-attention layer between the self-attention and cross-attention layers within the encoder. The details of the gated self-attention are depicted in the left part of the following figure.

**Comment by tuanh123789** [comment]
> @tuanh123789 hello, I wonder that the netework of GLIGEN is different from the diffusers and how to convert it successfully? As far as i know, it added new parameters in that, after converted, how to use them in diffusers?
> 
> > GLIGEN: The way GLIGEN introduces new parameters is by adding a gated self-attention layer between the self-attention and cross-attention layers within the encoder. The details of the gated self-attention are depicted in the left part of the following figure.

You can check this: https://huggingface.co/anhnct/Gligen_Text_Image

**Comment by zengjie617789** [comment]
Thanks yout shared link and helped me a lot. Furthermore, is it the diffusers deployment of GLIGEN?  Is each model same from the officel model which is based on SD1.4? And is it possible to change the unet or vae to another when i load that?
I will be appreciate if any help provided.

**Comment by tuanh123789** [comment]
> Thanks yout shared link and helped me a lot. Furthermore, is it the diffusers deployment of GLIGEN? Is each model same from the officel model which is based on SD1.4? And is it possible to change the unet or vae to another when i load that? I will be appreciate if any help provided.

Yes, is the version using in diffusers of GLIGEN. The weights of model is converted from offical checkpoint by me :D :D :D
